ballerina/tests/jwt_issuer_test.bal

```diff
@@ -311,6 +311,81 @@ isolated function testIssueJwtWithSigningAlgorithmRS512() {
     }
 }
 
+@test:Config {}
+isolated function testIssueJwtWithSigningAlgorithmHS256() {
+    IssuerConfig issuerConfig = {
+        username: "John",
+        issuer: "wso2",
+        audience: ["ballerina", "ballerinaSamples"],
+        expTime: 600,
+        signatureConfig: {
+            algorithm: HS256,
+            config: "s3cr3t"
+        }
+    };
+
+    string|Error result = issue(issuerConfig);
+    if (result is string) {
+        test:assertTrue(result.startsWith("eyJhbGciOiJIUzI1NiIsICJ0eXAiOiJKV1QifQ."));
+        string header = "{\"alg\":\"HS256\", \"typ\":\"JWT\"}";
+        string payload = "{\"iss\":\"wso2\", \"sub\":\"John\", \"aud\":[\"ballerina\", \"ballerinaSamples\"]";
+        assertDecodedJwt(result, header, payload);
+    } else {
+        string? errMsg = result.message();
+        test:assertFail(msg = errMsg is string ? errMsg : "Error in generated JWT.");
+    }
+}
```

Comment:
```suggestion
@test:Config {}
isolated function testIssueJwtWithSigningAlgorithmHS256() returns error? {
    IssuerConfig issuerConfig = {
        username: "John",
        issuer: "wso2",
        audience: ["ballerina", "ballerinaSamples"],
        expTime: 600,
        signatureConfig: {
            algorithm: HS256,
            config: "s3cr3t"
        }
    };

    string result = check issue(issuerConfig);
     test:assertTrue(result.startsWith("eyJhbGciOiJIUzI1NiIsICJ0eXAiOiJKV1QifQ."));
     string header = "{\"alg\":\"HS256\", \"typ\":\"JWT\"}";
     string payload = "{\"iss\":\"wso2\", \"sub\":\"John\", \"aud\":[\"ballerina\", \"ballerinaSamples\"]";
     assertDecodedJwt(result, header, payload);
}
```

Shall we use this pattern in other places as well ?

Reply:
@ayeshLK Thanks for pointing out these. I will make sure to update the complete codebase along with https://github.com/ballerina-platform/ballerina-standard-library/issues/1718. Same concern https://github.com/ballerina-platform/module-ballerina-jwt/pull/307#discussion_r684924378.